guardian/VaultDoor
# Fix / only download media not proxies etc.

## What does this change?
When retrieving the item synopsis list, only include rushes not other media types. All other search operations are unaffected

## How to test

- Use the "Project browser" function to find a project that has got metadata/proxy files as well as rushes (as shown by the pie-chart)
- Download using Download Manager. You should not get any VX-xxxxx files or extraneous xml files.

## How can we measure success?

Less pointless media copies

## Have we considered potential risks?

None
